### PR TITLE
Update free credit list with Alamy changes

### PR DESCRIPTION
**Pull request by kenoir**
Adds "Alamy Stock Photo" to freeCreditList

### CODE REVIEW

**Comment by jamesgorrie** [comment]
:+1:

File: common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala [hunk 1]
```diff
@@ -81,6 +81,9 @@ object AlamyParser extends ImageProcessor {
     case Some("Alamy") => image.copy(
       usageRights = Agency("Alamy")
     )
+    case Some("Alamy Stock Photo") => image.copy(
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala]: Or `case Some("Alamy") | Some("Alamy Stock Photo")`

**Comment by jamesgorrie** [comment]
@theefer updated and added test.

**Comment by theefer** [comment]
:+1:

**Comment by theefer** [comment]
Remember to re-index matching images so they acquire the correct rights and tell Jo.

**Comment by jamesgorrie** [comment]
Yip - will do.

**Pull request merged**